packages/grafana-ui/src/components/Logs/LogRowMessage.tsx

```diff
@@ -34,8 +33,8 @@ interface Props extends Themeable {
   updateLimit?: () => void;
 }
 
-const getStyles = stylesFactory((theme: GrafanaTheme) => {
-  const outlineColor = tinycolor(theme.colors.dashboardBg).setAlpha(0.7).toRgbString();
+const getStyles = (theme: GrafanaTheme2) => {
+  const outlineColor = tinycolor(theme.v1.colors.dashboardBg).setAlpha(0.7).toRgbString();
```

Comment:
this is a very strange, color is it used as a border? If so maybe use border.weak ?

Reply:
@torkelo yeah it seems to be used as colour for the border. I'll try what you suggested